src/main/haskell/kore/src/Kore/Substitution/Class.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE TypeApplications #-}
```

Comment:
`TypeApplications` is already enabled project-wide because it changes the behavior of GHC's parser.